libs/ui/src/lib/treemap-chart/treemap-chart.component.ts

```diff
@@ -211,4 +216,28 @@ export class GfTreemapChartComponent
 
     this.isLoading = false;
   }
+  private getTooltipPluginConfiguration() {
+    return {
+      ...getTooltipOptions({
+        colorScheme: this.colorScheme,
+        currency: this.baseCurrency,
+        locale: this.locale
+      }),
+      callbacks: {
+        label: (context) => {
+          const value = <number>context.raw._data.valueInBaseCurrency;
+
+          return `${value.toLocaleString(this.locale, {
```

Comment:
Can you fallback to `allocationInPercentage` (and show % as unit) if valueInBaseCurrency is `null`?

Reply:
Done!

![image](https://github.com/user-attachments/assets/3a4bdd53-2958-40e6-b892-95a8dc0ce048)